masterlist.yaml

```
@@ -29006,3 +29006,9 @@ plugins:
     clean:
       - crc: 0xC5B99C7E
         util: 'SSEEdit v4.0.4'
+
+  - name: 'mpThievesNightOnTheTown.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/42069'
+        name: "Thieves' Night On the Town -- Tnott"
```

The link just needs a trailing slash, & we prefer to escape single quotes in strings rather than use double quotes:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/42069/'
        name: 'Thieves'' Night On the Town -- Tnott'
```